src/WallpaperEngine/Render/CWallpaper.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <assert.h>
+
+#include "WallpaperEngine/Core/CWallpaper.h"
+
+namespace WallpaperEngine::Render
+{
+    class CWallpaper
```

Comment:
As I've explained earlier, this would be better inheriting irr::scene::ISceneNode to make use of the render function and be properly bound to the viewport limits, thus drawing the video only on the screends needed.